crates/matrix-sdk-ui/src/room_list/mod.rs

```diff
@@ -63,6 +63,7 @@
 //! machine's state, which can be pretty helpful for the client app.
 
 mod room;
+#[allow(clippy::module_inception)]
```

Comment:
You're not making clippy happy, you're silencing its complaints! 😅

But really, I agree with clippy here. `room_list::room_list` is weird.

Reply:
But I _am_ weird.

Reply:
I'll rename the directory from `room_list` to `room_list_service` in another PR. Just don't want to mess with this one too much.